### PR TITLE
Add script supporting cleaning up nightly images

### PR DESCRIPTION
**Pull request by kravvcuCL**
The script generates a list of tags which can be removed
based only on how many recent tags need to be preserved.

### CODE REVIEW

**Review by micczu1 — APPROVED** [review]

**Review by jabielecki — COMMENTED** [review]
I'm concerned with naming of a directory. Now we have `nexus_cleanup` directory, so maybe `dockerregistry_cleanup` instead of `nightly_images_cleanup`?

File: tungsten_ci_utils/nightly_images_cleanup/get_public_tags.sh [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+      ;;
+  esac
+done
+
+timestamp=$(date '+%Y-%m-%d-%H-%M-%S')
+tag_file="/tmp/tags-public-$timestamp"
+
+# get all tags from the registry
+. venv/bin/activate
+python docker_tagtool.py --registry ${REGISTRY} list_tags > ${tag_file}
```
Comment (jabielecki) [inline tungsten_ci_utils/nightly_images_cleanup/get_public_tags.sh]: No such file docker_tagtool.py exists

**Comment by kravvcuCL** [comment]
> I'm concerned with naming of a directory. Now we have `nexus_cleanup` directory, so maybe `dockerregistry_cleanup` instead of `nightly_images_cleanup`?

@jabielecki don't see the `nexus_cleanup` dir anywhere, but I renamed mine anyways.

File: tungsten_ci_utils/nightly_images_cleanup/get_public_tags.sh (hunk 1 above)
Comment (kravvcuCL) [reply tungsten_ci_utils/nightly_images_cleanup/get_public_tags.sh]: Added.

**Pull request merged**